x-pack/plugins/alerting/server/application/rule/methods/bulk_edit/bulk_edit_rules.ts

```diff
@@ -458,20 +472,15 @@ async function updateRuleAttributesAndParamsInMemory<Params extends RuleParams>(
       rule.references = migratedActions.resultedReferences;
     }
 
-    const ruleActions = injectReferencesIntoActions(
```

Comment:
`transformRuleAttributesToRuleDomain` calls `transformRawActionsToDomainActions` which in turn injects the references into the actions. No need to do it twice.